auto_round/wrapper.py

```diff
@@ -601,7 +601,7 @@ def __init__(self, orig_layer, bit=4, group_size=-1, device="cpu"):
     def unwrapper(self, best_params):
         if best_params is None:
             return self.orig_layer
-        v = best_params["v"]
+        v = best_params["v"].to(self.device)
```

Comment:
have not do the same for the  scale parameters, is it correct?

Reply:
In my check, only v is used in the best params